### PR TITLE
Add CI workflow so debian package will be built

### PR DESCRIPTION
**Pull request by maseabunikie**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: main
+
+on:
+  push:
+    branches: [ main ]
```
Comment (kulmesa) [inline .github/workflows/main.yaml]: should be master

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  pull_request:
+    branches: [ main ]
```
Comment (kulmesa) [inline .github/workflows/main.yaml]: should be master

File: .github/workflows/main.yaml (hunk 1 above)
Comment (maseabunikie) [reply .github/workflows/main.yaml]: Changed to master.

File: .github/workflows/main.yaml (hunk 2 above)
Comment (maseabunikie) [reply .github/workflows/main.yaml]: Changed to master. Thanks!

**Review by kulmesa — APPROVED** [review]

**Pull request merged**